tools/rln_keystore_generator/rln_keystore_generator.nim

```diff
@@ -24,7 +29,48 @@ when isMainModule:
   
   debug "configuration", conf = $conf
 
-  # initialize keystore
+  # 2. initialize rlnInstance
+  let rlnInstanceRes = createRLNInstance(d=20, 
+                                         tree_path = genTempPath("rln_tree", "rln_keystore_generator"))
+  if rlnInstanceRes.isErr():
+    error "failure while creating RLN instance", error=rlnInstanceRes.error()
```

Comment:
I think we tend not to use `()` for "name" attributes, although is being used in many places in this file
```suggestion
    error "failure while creating RLN instance", error=rlnInstanceRes.error
```

Reply:
Addressed in fed49b55